assets/js/components/notifications/IdeaHubModuleNotification.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * IdeaHubModuleNotification component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_MODULES } from '../../googlesitekit/modules/datastore/constants';
+import { CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { MODULES_IDEA_HUB } from '../../modules/idea-hub/datastore/constants';
+import Notification from '../legacy-notifications/notification';
+import IdeaHubNotificationSVG from '../../../svg/idea-hub-notification.svg';
+const { useSelect, useDispatch } = Data;
+
+const NOTIFICATION_ID = 'idea-hub-module-notification';
+
+const IdeaHubModuleNotification = () => {
+	const { dismissItem } = useDispatch( CORE_USER );
+
+	const isActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'idea-hub' ) );
+	const isItemDismissed = useSelect( ( select ) => select( CORE_USER ).isItemDismissed( NOTIFICATION_ID ) );
+	const adminReauthURL = useSelect( ( select ) => select( MODULES_IDEA_HUB )?.getAdminReauthURL() );
+
+	const handleOnDismiss = useCallback( async () => {
+		await dismissItem( NOTIFICATION_ID );
+	}, [ dismissItem ] );
+
+	if ( isActive || isItemDismissed || isItemDismissed === undefined ) {
```

Comment:
Technically, can't [`isActive` also be in an `undefined`/"loading" state](https://github.com/google/site-kit-wp/blob/65e6954e56d56167b15e218146e0e93e6ed852e5/assets/js/googlesitekit/modules/datastore/modules.js#L677-L679)? Similar to `isItemDismissed`, we might want to render `null` if it's still loading that info as well.

Reply:
👍